project_hr/tests/test_project_hr.py

```diff
@@ -59,6 +59,7 @@ def test_task(self):
             'user_id': self.user_2.id,
             'category_ids': [(6, 0, self.hr_category.ids)],
         })
+        self.task.invalidate_cache()
```

Comment:
You are invalidating cache here, but you should do it when creating the employee on regular code. If not, on a production database, computed field is not going to be refreshed.